salt/modules/virt.py

```diff
@@ -2265,9 +2265,9 @@ def update(
         Refer to :ref:`init-boot-def` for the complete boot parameter description.
 
         To update any boot parameters, specify the new path for each. To remove any boot parameters,
-        pass a None object, for instance: 'kernel': ``None``.
+        pass null object, for instance: 'kernel': ``null``.
```

Comment:
Don't mention `null` in that documentation since this is a YAML/SLS-specific value for `None`